nconnect.go

```diff
@@ -245,234 +244,271 @@ func Run(opts *config.Opts) {
 		ssConfig.UDPSocks = true
 	}
 
-	var tun *tunnel.Tunnel
+	nc := &nconnect{opts: opts, account: account, clientConfig: clientConfig, tunnelConfig: tunnelConfig, ssConfig: ssConfig,
```

Comment:
Line too long, better to break it, one field per line. See ssConfig at L233

Reply:
Yes, modified.